api/edit_observation.js

```diff
@@ -0,0 +1,28 @@
+var errors = require('../errors')
+var through = require('through2')
+
+module.exports = function (osm) {
+  return function () {
+    return through.obj(function (doc, _, cb) {
+      console.log(doc)
+      if (!doc.version) return cb(null, doc)
+      console.log('getting by version', doc.version)
+      osm.getByVersion(doc.version, function (err, element) {
+        if (err && (/^notfound/.test(err) || err.notFound)) {
+          err = errors(404, err)
+        }
+        if (err) return cb(err)
+        if (doc.type === 'observation') {
+          var obs = Object.assign({}, doc, { type: 'observation' })
+          obs.attachments = element.attachments
+          obs.metadata = element.metadata
+          obs.schemaVersion = element.schemaVersion
+          return cb(null, doc)
```

Comment:
Did you mean to do:
```suggestion
          return cb(null, obs)
```
?